ansible/roles_ocp_workloads/ocp4_workload_quarkus_workshop_user/tasks/pre_workload.yml

```diff
@@ -66,6 +66,7 @@
     - "console URL: https://{{ r_console_route.resources[0].spec.host }}"
     - "route subdomain: {{ route_subdomain }}"
     - "ocp_username: {{ ocp_username }}"
+    - "guid: {{ guid }}"
```

Comment:
don't you want `agnosticd_user_info` instead of `debug` here, so the info is sent up to the user ?

Reply:
these aren't for the user - only debug / deploy logs in case something goes wrong... I have `agnosticd_user_info` later on my post workload tasks.